### PR TITLE
Screenreaders new tab disclosure

### LINKED ISSUE #1855: The external link should disclose to the screenreaders that it opens in a new tab
## Problem

Get this audio link opens in a new tab, but does not disclose this to screen readers.

## Description

The SVG 'external link' graphic needs an accessible name that notifies the user that the link opens in a new tab.

## Additional context

This was originally reported for the `Get audio` external link, by Joe Dolson (https://make.wordpress.org/openverse/2022/09/07/frontend-release-v3-4-8-and-a-call-for-a11y-testing/#comment-95)

## Implementation

- [ ] 🙋 I would be interested in implementing this feature.

### PR DESCRIPTION
**Pull request by AlexVCS**
## Fixes
Fixes #1855 

## Description
Adds span tag to VLink.vue and aria-new-tab value to en.json5

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no visible errors.
       ⬆️ Currently I'm getting an error, but I don't think it relates to any of the changes I made.
       image

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: src/locales/scripts/en.json5 [hunk 1]
```diff
@@ -18,6 +18,7 @@
       label: 'I want something I can',
     },
     aria: {
+      "aria-new-tab": "(opens in a new tab)"
```
Comment (obulat) [inline src/locales/scripts/en.json5:21]:
```suggestion
      "aria-new-tab": "(opens in a new tab)",
```

File: src/components/VLink.vue [hunk 2]
```diff
@@ -26,7 +26,8 @@
       :size="4"
       rtl-flip
     />
-  </a>
+    <span class="sr-only">{$t('aria-new-tab')</span>
```
Comment (obulat) [inline src/components/VLink.vue:29]:
```suggestion
    <span class="sr-only">{{ $t('aria-new-tab') }}</span>
```

**Comment by AlexVCS** [comment]
I pulled in the changes you made and I also ran 

```pnpm i18n:watch-en```

This is the error message I get when I run that command:

Screen Shot 2022-11-24 at 4 50 24 PM

 I still can't get the application running, despite my attempts. Any idea what may be wrong and where I should look to fix it? Thanks

**Comment by obulat** [comment]
I rebased this PR, too, @AlexVCS. Could you please try running `pnpm dev` here, too? If you still see errors, maybe you could delete node_modules and re-run `pnpm install`, and then try `pnpm dev` once again?

**Comment by obulat** [comment]
> I still can't get the application running, despite my attempts. Any idea what may be wrong and where I should look to fix it? Thanks

I've noticed that your `main` branch is quite old, its last updated date is more than 3 months ago. This might be the cause for the errors you're getting, @AlexVCS. Please let me know if you need help with updating it.

**Comment by krysal** [comment]
@AlexVCS Are you still interested in finishing this work?

**Comment by AlexVCS** [comment]
> @AlexVCS Are you still interested in finishing this work? Just a check-in! It's okay if you don't the capacity right now but we would appreciate you let us know :)

really sorry to respond late! I see that Olga has been making some commits, but if there is anything needed still I would be happy to help

**Comment by zackkrida** [comment]
@AlexVCS and @obulat, do you think we should mark this "Ready for Review"?

**Comment by obulat** [comment]
I am going to close this PR because the simple change is causing line breaks in the snapshots between words and punctuation marks, and the larger changes have grown too much to safely merge.

**Pull request closed without merging**